### DISCUSSION TITLE
BTicino FC80GT

### DISCUSSION THREAD

**Discussion by Solarflor** [FredBlo/node-red-contrib-myhome-bticino-v2#14] [General]
Hi,
is there any way to control such Gateway with Node-Red?
Thanks
Floriano

**Comment by FredBlo** [comment]
Hi @Solarflor ,

I searched a bit about the FC80GT gateway.
Compared to 'MyHome' this flow covers, it is another protocol-based (Netatmo), so this flow will certainly not cover your needs.

The question is more related to node-red itself, I would search on available flows using Netatmo and see if any can cover what you need to achieve.
https://flows.nodered.org/search?term=Netatmo

Regards

Fred

**Reply by Solarflor** [reply]
Many thanks FredBlo
